decidim-core/app/views/layouts/decidim/_social_meta.html.erb

```diff
@@ -0,0 +1,11 @@
+<meta name="twitter:card" content="summary_large_image" />
+<meta name="twitter:site" content="@<%= content_for(:twitter_handler) %>" />
+<meta name="twitter:title" content="<%= content_for(:proposal_title) %>" />
```

Comment:
You're probably going ot kill me but: I think those proposal_title, proposal_description, etc should be something like social_title or meta_title, there's more stuff than just proposals.